### PR TITLE
Roadmap project: One Logging Bridge per Language

### LINKED ISSUE #1865: One Logging Bridge per Language
@jpkrohling has volunteered to be the sponsor for this.

### PR DESCRIPTION
**Pull request by jpkrohling**
Closes #1865

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: projects/one-logging-bridge-per-language.md
Comment (pellared) [inline projects/one-logging-bridge-per-language.md:3]:
```suggestion
During the joint TC/GC call on Jan 11 2024, we agreed on a few items we'd like to accomplish this year. One such item was creating at least one logging bridge per Language SIG, so that our end-users can start using OTLP Logging natively in their applications.
```

**Review by pellared — APPROVED** [review]

**Review by srikanthccv — APPROVED** [review]
I think swift could also be added to the list

File: projects/one-logging-bridge-per-language.md
Comment (svrnm) [inline projects/one-logging-bridge-per-language.md:26]:
```suggestion
- [ ] One Logging Bridge per language - Swift
```

**Review by cijothomas — APPROVED** [review]

**Review by svrnm — APPROVED** [review]

**Review by trask — APPROVED** [review]

**Comment by jpkrohling** [comment]
Could another maintainer of this repo please merge this, if we have enough reviews?

**Pull request merged**
